packages/local_auth/local_auth/lib/error_codes.dart

```diff
@@ -9,18 +9,18 @@
 /// PIN/pattern/password (Android) on the device.
 const String passcodeNotSet = 'PasscodeNotSet';
 
-/// Indicates the user has not enrolled any fingerprints on the device.
+/// Indicates the user has not enrolled any biometrics on the device.
 const String notEnrolled = 'NotEnrolled';
 
-/// Indicates the device does not have a Touch ID/fingerprint scanner.
+/// Indicates the device does not have hardware support for biometrics.
```

Comment:
But previously we use `getAvailableBiometrics` to refer to enrolled biometrics. Maybe "NoBiometricHardware"?